src/Identity/testassets/Identity.DefaultUI.WebSite/StartupBase.cs

```diff
@@ -40,11 +40,11 @@ public virtual void ConfigureServices(IServiceCollection services)
             services.AddDbContext<TContext>(options =>
                 options
                     .ConfigureWarnings(b => b.Log(CoreEventId.ManyServiceProvidersCreatedWarning))
-                    //.UseSqlServer(
-                    //    Configuration.GetConnectionString("DefaultConnection"),
-                    //    sqlOptions => sqlOptions.MigrationsAssembly("Identity.DefaultUI.WebSite")
-                    //));
-                    .UseSqlite("DataSource=:memory:"));
+                    .UseSqlServer(
+                        Configuration.GetConnectionString("DefaultConnection"),
+                        sqlOptions => sqlOptions.MigrationsAssembly("Identity.DefaultUI.WebSite")
+                    ));
+            //.UseSqlite("DataSource=:memory:"));
```

Comment:
Not sure about the impact of this change of if it was a "bug" in the old tests.

Reply:
Reverting this was just for local debugging thanks